app/javascript/app/data/initial-state.js

```diff
@@ -1,10 +1,238 @@
 export default {
+  nav: {
+    countriesOpen: false
+  },
+  map: {
+    center: [0, 20],
+    zoom: 1
+  },
   countryExplorer: {
     open: false
   },
   countryNDC: {
     loading: false,
     loaded: false,
     data: {}
+  },
+  countrySelect: {
+    query: '',
+    countries: [
```

Comment:
Agree and I like it more, so I'll create and export the chopped part inside the component-reducer.